be/test/CMakeLists.txt

```diff
@@ -510,6 +510,8 @@ set(EXEC_FILES
 if ("${WITH_STARCACHE}" STREQUAL "ON")
     list(APPEND EXEC_FILES ./cache/block_cache/block_cache_test.cpp)
     list(APPEND EXEC_FILES ./cache/block_cache/disk_space_monitor_test.cpp)
+    list(APPEND EXEC_FILES ./cache/object_cache/starcache_module_test.cpp)
+    list(APPEND EXEC_FILES ./cache/object_cache/object_cache_test.cpp)
```

Comment:
object cache test cases should be run either with starcache module or lrucache module, not only with starcache_module.

Reply:
I will fix it later pr